### DISCUSSION TITLE
EF-like bundes

### DISCUSSION THREAD

**Discussion by BISmb** [fluentmigrator/fluentmigrator#1660] [Ideas]
I think it would be awesome to implement bundles similar to how entity framework has setup. This would enable database-agnostic CI/CD besides using `dotnet-fm` on the project.

https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/applying?tabs=dotnet-core-cli#bundles

**Comment by jzabroski** [comment]
I'll look into it but I am skeptical this is very valuable.  Just because Microsoft did something doesn't make it awesome.

I have had my own ideas for how to support database-agnostic CI/CD.  But I will take your advice seriously and try to study how exactly efBundles feature works.  Do you know the original GitHub Issue it is linked to?  I dont think the URL you provided is a detailed explanation of it.

**Reply by jzabroski** [reply]
https://devblogs.microsoft.com/dotnet/introducing-devops-friendly-ef-core-migration-bundles/#introducing-migration-bundles

Got it. The key feature of bundles is that .net core global cli tools are not intended to be run in production environments, since it requires installing the SDK.

I think we can do better than EF bundles.
